docs/user/dql/functions.rst

```diff
@@ -1219,15 +1219,60 @@ Example::
     +---------------------------+
 
 
+MAKEDATE
+--------
+
+Description
+>>>>>>>>>>>
+
+Returns a date, given `year` and `day-of-year` values. `dayofyear` must be greater than 0 or the result is `NULL`. The result is also `NULL` if either argument is `NULL`.
+Arguments are rounded to an integer.
+
+Specifications:
+
+1. MAKEDATE(DOUBLE, DOUBLE) -> DATE
+
+Argument type: DOUBLE
+
+Return type: DATE
+
+Example::
+
+    os> select MAKEDATE(1945, 5.9) as f1, MAKEDATE(1984, 1984) as f2
+    fetched rows / total rows = 1/1
+    +------------+------------+
+    | f1         | f2         |
+    |------------+------------|
+    | 1945-01-06 | 1989-06-06 |
```

Comment:
If sample code is going to show that `MAKEDATE(1984, 1984) is 1989-...` the description should describe this behaviour as well.

This is also a good test case to have in the unit tests -- that `MAKEDATE($year, $days_in_year + 1) == MAKEDATE($year + 1, 1)`.